firebase-firestore/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Unreleased
 * [fixed] Fixed the `null` value handling in `whereNotEqualTo` and `whereNotIn` filters.
+* [fixed] Catch exception when stream is already cancelled during close. [#6894](//github.com/firebase/firebase-android-sdk/pull/6894)
```

Comment:
Suggest rewording this changelog entry to call out the issue that the customer would experience that is fixed, maybe something like

> Fix intermittent IllegalStateException: "call was cancelled" when closing Firestore